src/ice_setup_step.F90

```diff
@@ -31,6 +31,7 @@ subroutine ice_setup(mesh)
     ice_dt=real(ice_ave_steps,WP)*dt
     ! ice_dt=dt
     Tevp_inv=3.0_WP/ice_dt 
+    Tevp_inv=1.0_WP/1800.0_WP
```

Comment:
That's a bit too radical :) We will have to make it a switch to be able to keep old behaviour.

Reply:
Hey @koldunovn! Agreed... I was just testing something on the branch and I forgot there was a pull request open associated with it! I will switch to a different branch and revert this change.

Reply:
Nice to see you back in the FESOM2 repo! :)

Reply:
Yes, I gave myself a couple of days to clean a bit of the mess I left behind ;)